packages/client/hmi-client/src/assets/css/theme/_variables.scss

```diff
@@ -502,7 +502,7 @@ $tableHeaderFontWeight: 500 !default;
 $tableHeaderPadding: 1rem 1rem !default;
 
 $tableHeaderCellPadding: 1rem 0.25rem 1rem !default;
-$tableHeaderCellBg: #ffffff !default;
+$tableHeaderCellBg: #F8F9FA !default;
```

Comment:
We do not have a (CSS variable)[https://github.com/DARPA-ASKEM/terarium/blob/49ca5a9e0720639b5a1bf7d4e274c3d52efd7bc6/packages/client/hmi-client/src/assets/css/theme/_variables.scss#L866] for it?

Reply:
should I add one?  There is no variable with that colour that I can see...

Reply:
Nah, this is up to the design team to come up with proper colour scheme.